Libs/Widgets/ctkMatrixWidget.cpp

```diff
@@ -238,7 +238,9 @@ void ctkMatrixWidgetPrivate::updateGeometries()
     bool lastColumn = (j==(ccount-1));
     int newWidth = lastColumn ? lastColWidth : colWidth;
     this->Table->setColumnWidth(j, newWidth);
-    Q_ASSERT(this->Table->columnWidth(j) == newWidth);
+    qWarning() << Q_FUNC_INFO << " failed:"
```

Comment:
the warning should be reported only with `if (this->Table->columnWidth(j) != newWidth)`

Reply:
we could also include the code within `#ifndef NDEBUG`, that way release build will not complain

Reply:
Good catch, thank you!
Maybe we should add a macro for this that we can use to replace Q_ASSERTs?